Logstash/logstash-input-azureblob/logstash-input-azureblob.gemspec

```diff
@@ -23,4 +24,10 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'stud', '>= 0.0.22'
   s.add_runtime_dependency 'azure-storage', '~> 0.12.3.preview'
   s.add_development_dependency 'logstash-devutils'
+  s.add_development_dependency 'logging'
+
+  # Jar dependencies
+  #s.requirements << "jar 'javax.json:javax.json-api', '1.1'"
+  s.requirements << "jar 'org.glassfish:javax.json', '1.1'"
```

Comment:
What's the new thing that this other parser can and the previous cannot?

Reply:
this is just an artifact. The commented line was just my first try in getting the JsonParser dependency (it was the wrong namespace). Previously we didn't have any parser, just a simple string check